docs/conf.py

```diff
@@ -115,7 +115,7 @@
     'extra_nav_links': OrderedDict((
         ('MDN', 'https://developer.mozilla.org'),
         ('MDN Staging', 'https://developer.allizom.org'),
-        ('Kuma on GitHub', 'https://github.com/mozilla/kuma'),
+        ('Kuma on GitHub', 'https://github.com/mdn/kuma'),mdn
```

Comment:
There's an extra 'mdn' appended on the change to 118?

Reply:
Pretty sad it slipped through without a pre-commit hook not catching it.